user-guide/Cloud_Platform/TeamsBot/Microsoft_Teams_Chat_Integration.md

```diff
@@ -2,57 +2,67 @@
+The following features are available:
 
-## Using Chat Integration
+- Creating teams
+- Creating channels
+- Adding members or owners to your teams
+- Sending notifications in the created channels in the name of the DataMiner Teams bot
+- Creating a private chat between someone and the DataMiner Teams bot
+- Fetching a private chat between someone and the DataMiner Teams bot
+- Sending notifications in those private chats in the name of the DataMiner Teams bot
 
-> [!TIP]
-> For examples of chat integration Automation scripts, refer to [Chat Integration Examples](https://github.com/SkylineCommunications/ChatOps-Extensions/tree/main/ChatIntegrationExamples) on GitHub.
+To get started, you can find several example Automation scripts with more information on [GitHub](https://github.com/SkylineCommunications/ChatOps-Extensions/tree/main/ChatIntegrationExamples).
+
+After you have made sure the above-mentioned prerequisites are in place, you can download and deploy [the DcpChatIntegrationExamples DMAPP](https://github.com/SkylineCommunications/ChatOps-Extensions/blob/main/ChatIntegrationExamples/README.md#getting-started) to your DataMiner System and immediately try out these examples.
 
 ## Limitations
 
-- Currently you can not fetch existing teams. You can only create a channel using chat integeration if you know the id of the team, which you'll receive when creating the team with chat integration. 
-- Currently you can not fetch existing channels. You can only send notifications in a channel using chat integeration if you know the id of the channel, which you'll receive when creating the channel with chat integration. 
+- Currently, it is not possible to retrieve the existing teams. You can only create a channel using chat integration if you know the ID of the team, which you will receive when you create the team with chat integration.
+- Currently, it is not possible to retrieve the existing channels. You can only send notifications in a channel using chat integration if you know the ID of the channel, which you will receive when you create the channel with chat integration.
 
 ## Security
 
-By granting [Admin consent](xref:Granting_admin_consent) from your Microsoft (Teams) Tenant to 'DataMiner', aka Skyline Communications, in the [Admin app](https://admin.dataminer.services), you allow Skyline Communications to do these chat integration actions, in example creating teams, channels, chats, fetching users, chats etc. This access can be revoked at any time as explained in the [Admin consent documentation](xref:Granting_admin_consent).
\ No newline at end of file
+By granting [Admin consent](xref:Granting_admin_consent) from your Microsoft (Teams) tenant to "DataMiner" (i.e. Skyline Communications) in the [Admin app](https://admin.dataminer.services), you allow Skyline Communications to execute these chat integration actions. This includes creating teams, channels, and chats, fetching users, chats, etc.
```

Comment:
Here we say **Skyline Communications**, on _Grant Admin consent_ page we just say **Skyline**, we should align it but i'm not sure to say just skyline or the full name